### PR TITLE
#73 implement futures python

### PR DESCRIPTION
**Pull request by jmp3833**
This will cause some UI hiccups until the Node story (step 3/3) is completed. Please do not merge until every step in the UI refactoring pipeline is complete :)

### CODE REVIEW

File: linguine/operation_builder.py [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
 from linguine.ops.remove_stopwords import RemoveStopwords
 from linguine.ops.sentence_tokenize import SentenceTokenize
-from linguine.ops.stem import StemmerLancaster, StemmerPorter, StemmerSnowball
+from linguine.ops.stem import StemmerPorter
 from linguine.ops.topic_model import TopicModel
 from linguine.ops.word_tokenize import WordTokenizeTreebank, WordTokenizeWhitespacePunct, WordTokenizeStanford, WordTokenizeSpaces, WordTokenizeTabs
 from linguine.ops.StanfordCoreNLP import StanfordCoreNLP
 
 def get_operation_handler(operation):
+    print(operation)
```
Comment (jmp3833) [inline linguine/operation_builder.py]: Redundant print statement

File: linguine/webserver.py [hunk 2]
```diff
@@ -18,12 +19,24 @@
...
+    analysis_executor = ThreadPoolExecutor(max_workers=5)
+
     def post(self):
         self.set_header('Content-Type', 'application/json')
         try:
             transaction = Transaction()
-            transaction.parse_json(self.request.body)
-            self.write(transaction.run())
+            requestObj = transaction.parse_json(self.request.body)
+            transaction.read_corpora(transaction.corpora_ids)
```
Comment (jmp3833) [inline linguine/webserver.py]: This could be incorporated into the run function to reduce overhead

**Comment by kmp3325** [comment]
Looks great here!

File: linguine/transaction.py [hunk 3]
```diff
@@ -8,20 +11,59 @@
...
         self.corpora_ids = []
+        self.time_created = None
         self.corpora = []
+        self.analysis_pool = Pool(processes=5)
         self.analysis_name = ""
         self.cleanups = []
+        self.current_result = None
         self.tokenizer = None
-
+        self.timestamp = ""
```
Comment (jmp3833) [inline linguine/transaction.py]: This variable is no longer used

**Pull request merged**